libkbfs/kbfs_ops_concur_test.go

```diff
@@ -1794,8 +1794,6 @@ func (booq *blockOpsOverQuota) Put(ctx context.Context, tlfID tlf.ID,
 // Test that a quota error causes deferred writes to error.
 // Regression test for KBFS-751.
 func TestKBFSOpsErrorOnBlockedWriteDuringSync(t *testing.T) {
-	t.Skip("Broken pending KBFS-1261")
-
```

Comment:
Since you're in here, can you do some anti-hanging changes? In particular:

- Buffer syncErrCh and writeErrCh with 1.
- add a select with `case <-ctx.Done(): t.Fatal(ctx.Err()` when blocking on `onSyncStalledCh`, `<-syncErrCh`, and `<-writeErrCh`